test/fast-list-edit-test.js

```diff
@@ -344,7 +346,7 @@ suite('FastList > Edit >', function() {
           function() {
             assert.equal(scheduler.feedback.callCount, 2);
 
-            var reset = list.querySelectorAll('li:not([data-tweak-delta=""])');
+            var reset = list.querySelectorAll('li[data-tweak-delta=""]');
```

Comment:
OK I've looked into this deeper. I'm actually really confused. The selector didn't seem to be returning what it should have been. I've swapped it for a more precise `.reduce()` method to pull out the items that have `dataset.tweakDelta` values and everything seems to work as expected.

Any ideas?